alethzero/DappLoader.cpp

```diff
@@ -96,13 +97,31 @@ DappLocation DappLoader::resolveAppUri(QString const& _uri)
 
 void DappLoader::downloadComplete(QNetworkReply* _reply)
 {
+	QUrl requestUrl = _reply->request().url();
+	if (m_pageUrls.count(requestUrl) != 0)
+	{
+		//inject web3 js
+		QByteArray content = "<script>\n";
+		content.append(web3Content());
+		content.append("</script>\n");
+		content.append(_reply->readAll());
```

Comment:
In that case, the script tag containing the web3 obj will be added outside of the HTML tag.
Can this cause side effect as this is not standard?